src/tools/crossgen2/Common/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -1079,7 +1080,7 @@ protected override void AllocManagedToNative(ILCodeStream codeStream)
             codeStream.Emit(ILOpcode.mul_ovf);
 
             codeStream.Emit(ILOpcode.call, emitter.NewToken(
```

Comment:
Just checking - is the ZeroMemory part in the original version really unnecessary?

Reply:
I convinced myself it's unnecessary - in the managed to native direction we always end up copying the elements from the managed side (unless this is `Out` marshalling, in which case we just marshal `null`).